### PR TITLE
feat(components): Add the column button icon to atlantis [JOB-63504]

### PR DESCRIPTION
**Pull request by kyleherrmann**
## Motivations

- So that the column selection feature in the new one-off jobs report can used the designed icon

### Added
- New icon and updated storybook

## Testing
 - Navigate to the [Other section](http://localhost:6005/?path=/docs/components-icon--icon#other) under icons and validate the `tableColumns` icon was correctly added

Screenshot 2023-04-20 at 3 24 12 PM

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/design/src/icons/iconMap.ts [hunk 1]
```diff
@@ -439,9 +450,6 @@ export const iconMap = {
     ],
     "today": [
       "M682.667 128c-23.565 0-42.667 19.103-42.667 42.667v42.667h-256v-42.667c0-23.564-19.103-42.667-42.667-42.667s-42.667 19.103-42.667 42.667v42.667h-85.333c-47.128 0-85.333 38.205-85.333 85.333v512c0 47.13 38.205 85.333 85.333 85.333h597.333c47.13 0 85.333-38.204 85.333-85.333v-512c0-47.128-38.204-85.333-85.333-85.333h-85.333v-42.667c0-23.564-19.102-42.667-42.667-42.667zM810.667 298.667v85.333h-597.333v-85.333h597.333zM213.333 469.333h597.333v341.333h-597.333v-341.333zM704 640c-35.345 0-64-28.655-64-64s28.655-64 64-64c35.345 0 64 28.655 64 64s-28.655 64-64 64z"
-    ],
-    "drag": [
```
Comment (kyleherrmann) [inline packages/design/src/icons/iconMap.ts:443]: This file was autogenerated, is this change expected? 🤔

File: packages/design/src/icons/iconMap.ts [hunk 2]
```diff
@@ -5,6 +5,17 @@
 export const iconMap = {
   "height": 1024,
   "icons": {
+    "tableColumns": [
+      "M810.667 170.667h-597.333c-47.128 0-85.333 38.205-85.333 85.333v512c0 47.13 38.205 85.333 85.333 85.333h597.333c47.13 0 85.333-38.204 85.333-85.333v-512c0-47.128-38.204-85.333-85.333-85.333zM256 256h85.333v512h-85.333c-23.564 0-42.667-19.102-42.667-42.667v-426.667c0-23.564 19.103-42.667 42.667-42.667zM426.667 768v-512h170.667v512h-170.667zM682.667 768v-512h85.333c23.565 0 42.667 19.103 42.667 42.667v426.667c0 23.565-19.102 42.667-42.667 42.667h-85.333z"
+    ],
+    "drag": [
```
Comment (kyleherrmann) [inline packages/design/src/icons/iconMap.ts:11]: Commented [here](https://github.com/GetJobber/atlantis/pull/1199/files#r1173143677).

File: packages/design/src/icons/iconMap.ts (hunk 1 above)
Comment (taylorvnoj) [reply packages/design/src/icons/iconMap.ts:443]: Yup this is ok - somehow this was added to the bottom of `iconMap.ts` when it should have been at the top.  Probably related to how it was exported at the time.
Either way your PR fixed that for us, so thank you 👍

**Review by taylorvnoj — APPROVED** [review]
LGTM 🚀

**Pull request merged**